src/components/TwoColumnLayout/index.js

```diff
@@ -0,0 +1,50 @@
+import { RichTypography } from "@commons-ui/core";
+import { Grid, Typography } from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import Section from "@/pesayetu/components/Section";
+
+const TwoColumnLayout = ({ content, title, subtitle, ...props }) => {
+  const classes = useStyles(props);
+  if (!content) {
+    return null;
+  }
+
+  return (
+    <div className={classes.root}>
+      <Section classes={{ root: classes.section }}>
+        <Grid container>
+          <Grid item xs={12} lg={4}>
+            <Typography className={classes.title} variant="h1">
+              {title}
+            </Typography>
+            <Typography className={classes.subtitle} variant="h4">
+              {subtitle}
+            </Typography>
+          </Grid>
+          <Grid item lg={2} />
```

Comment:
If we need to skip columns **_between_** two `Grid item`s, wouldn't justifyContent="space-between" be enough?